### PR TITLE
feature/DEVOPS 174 deprecation upgrades

### PR DESCRIPTION
**Pull request by hero-david**
- chore: convert set_output gem commands to use GITHUB_OUTPUT file
- :arrow_up: chore: update to latest helm and slack action versions  - helm: v1 => v3  - slack: v1.19 => v1.23

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -46,9 +46,9 @@ jobs:
         env:
           BUILD: ${{ github.run_number }}
         run: |
-          echo "::set-output name=build_num::$(($BUILD+10000))"
-          echo "::set-output name=image_tag_build::${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:$(($BUILD+10000))"
-          echo "::set-output name=image_tag_branch::${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:${{github.ref_name}}"
+          echo "build_num=$(($BUILD+10000))"
+          echo "image_tag_build=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:$(($BUILD+10000))"
+          echo "image_tag_branch=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:${{github.ref_name}}"
```
Comment (matthewhartstonge) [inline .github/workflows/php.yml:51]:
```suggestion
          echo "build_num=$(($BUILD+10000))" >> $GITHUB_OUTPUT
          echo "image_tag_build=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:$(($BUILD+10000))" >> $GITHUB_OUTPUT
          echo "image_tag_branch=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:${{github.ref_name}}" >> $GITHUB_OUTPUT
```

File: .github/workflows/php.yml [hunk 2]
```diff
@@ -103,9 +103,9 @@ jobs:
...
```
Comment (matthewhartstonge) [inline .github/workflows/php.yml:108]:
```suggestion
          echo "build_num=$(($BUILD+10000))" >> $GITHUB_OUTPUT
          echo "image_tag_build=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:$(($BUILD+10000))" >> $GITHUB_OUTPUT
          echo "image_tag_branch=${{env.CONTAINER_REGISTRY}}/${{ inputs.CONTAINER }}-${{matrix.container}}:${{github.ref_name}}" >> $GITHUB_OUTPUT
```

**Review by matthewhartstonge — CHANGES_REQUESTED** [review]
Looks like you missed pushing the echo'd statements out to Github output for PHP 😱

**Comment by hero-david** [comment]
Balls! :sob: Thanks for the catch there, it was a bit of a mindless excercise

**Review by matthewhartstonge — APPROVED** [review]
LGTM!

**Pull request merged**